### PR TITLE
Added Hyperlinks to Brand Assets and Contribution Guideline

### PR DESCRIPTION
**Pull request by SahasR**
Added Hyperlinks to Brand Assets and Contribution Guideline into the footer.

### CODE REVIEW

**Comment by ebonynon** [comment]
LGMT

File: src/app/components/common/footer/footer.component.html [hunk 1]
```diff
@@ -26,10 +26,15 @@ <h3 class="footer-heading">Helpful Links</h3>
...
+            <a
+              href="https://github.com/sliit-foss/sliitfoss/blob/master/README.md"
+              >Contribution Guideline</a
+            >
           </li>
           <li>
-            <a href="#">Brand Assets</a>
+            <a href="https://github.com/sliit-foss/brand-assets"
+              >Brand Assets</a
+            >
```
Comment (Shehanka) [inline src/app/components/common/footer/footer.component.html:37]: target="_blank"

**Review by Shehanka — CHANGES_REQUESTED** [review]
Do the changes I requested

**Review by Shehanka — APPROVED** [review]
good job

**Pull request merged**